### PR TITLE
Adding the see other themes link

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey Joram have a look.

### CODE REVIEW

**Comment by jwilander** [comment]
+1

**Comment by crspeller** [comment]
@asaadmahmoodspin Can you close this on and open another please? A build was not performed because I didn't have jenkens setup to build release branch PRs.

**Comment by crspeller** [comment]
Oh ya... the build completely changed....

**Comment by asaadmahmoodspin** [comment]
Not sure how to fix this. Illegal import declaration.
screen shot 2016-03-31 at 6 07 40 am

**Comment by jwilander** [comment]
It needs to relative, so just change the lines back to

```
import * as Utils from '../../utils/utils.jsx';
import Constants from '../../utils/constants.jsx';
```

File: web/react/components/user_settings/premade_theme_chooser.jsx [hunk 1]
```diff
@@ -4,6 +4,10 @@
 import * as Utils from '../../utils/utils.jsx';
 import Constants from '../../utils/constants.jsx';
 
+import React from 'react';
```
Comment (crspeller) [inline web/react/components/user_settings/premade_theme_chooser.jsx]: We don't have to do this on the 2.1 branch.

**Comment by crspeller** [comment]
@asaadmahmoodspin I won't be merging this branch back into master, you will need to submit this feature to master as well.

**Pull request merged**